src/eureka/S5_lightcurve_fitting/plots_s5.py

```diff
@@ -109,19 +109,19 @@ def plot_fit(lc, model, meta, fitter, isTitle=True):
         if isTitle:
             ax[0].set_title(f'{meta.eventlabel} - Channel {channel} - '
                             f'{fitter}')
-        ax[0].set_ylabel('Normalized Flux', size=14)
+        ax[0].set_ylabel('Normalized Flux', size=10)
```

Comment:
These are too small and will be much too small when using LaTeX fonts. I'll have to think a bit as to how to better fix these figures, but for now please reset these back to 14

Reply:
This stack overflow might help (if we choose to address this)? https://stackoverflow.com/questions/33942210/consistent-fonts-between-matplotlib-and-latex

Reply:
So I use LaTeX fonts most of the time, but my understanding is that they only work if you've installed LaTeX on your computer which is a lengthy and annoying process. It's possible matplotlib is able to do LaTeX fonts without LaTeX installed, but not from my experience. Additionally, using LaTeX fonts substantially slows figure creation, so when I fit NIR data with hundreds of channels I turn off LaTeX fonts to save like 50% of the runtime. Either way, I think this is something that'll take a moderate amount of fiddling on my part to get just right (it's definitely on my to-do list though)